tests/TestCase/View/Helper/TimeHelperTest.php

```diff
@@ -513,4 +520,21 @@ public function testNullDateFormat()
         $result = $this->Time->format(null, \IntlDateFormatter::FULL, $fallback);
         $this->assertEquals($fallback, $result);
     }
+
+    /**
+     * Test config options
+     */
+    public function testConfigOptions()
+    {
+        $Time = new TimeHelper($this->View);
+        $this->assertEquals($Time->config('defaultLocale'), null);
+        $this->assertEquals($Time->config('defaultFormat'), null);
+        $this->assertEquals($Time->config('defaultOutputTimezone'), null);
```

Comment:
You should use `assertNull()` when checking for null. `assertEquals()` will pass on false, and ''.